frontend/src/resources/utils/resource-request.ts

```diff
@@ -61,6 +62,37 @@ export function createResource<Resource extends IResource, ResultType = Resource
     return postRequest<Resource, ResultType>(url, resource)
 }
 
+export async function createResources(resources: IResource[]): Promise<string | undefined> {
+    if (!Array.isArray(resources)) return 'Error - resources are not an array'
+    for (const resource of resources) {
+        try {
+            const existingResource = await getResource(resource).promise
+            if (existingResource)
+                return 'Resource of kind {0} in namespace {1} already exists'
+                    .replace('{0}', resource.metadata?.name ?? '')
+                    .replace('{1}', resource.metadata?.namespace ?? '')
```

Comment:
need to filter the existing resource from `resources` so we don't try to create it below.

Reply:
This is for the create case and we return an error string if any already exist.  Will need to adjust for the case of update existing.